scripts/slimy-update-snap.sh

```diff
@@ -71,8 +72,14 @@ rm -rf "${subiquity_dest}/subiquity"
 rm -rf "${subiquity_dest}/subiquitycore"
 
 (cd "${src}" && ./scripts/update-part.py curtin)
+(cd "${src}" && ./scripts/update-part.py probert)
+
+# Build probert for the C extensions
+probert_tmpdir=$(mktemp -d)
```

Comment:
I would like to see this as a subdir of `tmpdir` which is covered by the handler registered with `trap`.

Reply:
I guess this comment doesn't apply anymore since I'm building the C extensions in place now.